test/aspect/aspect.bzl

```diff
@@ -23,32 +23,32 @@ def _rule_impl(ctx):
   expected_deps = {
       "scala_library": [
           "//test/aspect:scala_library",
-          "@scala_default_imports//:scala-library",
+          "@io_bazel_rules_scala_scala_library_2_11_11//:io_bazel_rules_scala_scala_library_2_11_11",
```

Comment:
you can and probably should replace with shorthand notation of @repo_name

Reply:
The full paths actually need to be here as the test does string comparison between this and what is returned by `test_aspect`